### PR TITLE
README - recommend to install `R.utils`

### PR DESCRIPTION
**Pull request by satai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -277,7 +277,7 @@ Install the following R packages:
 To install these packages from within an R session:
 
 ```r
-pkgs <- c("RCurl", "jsonlite", "statmod", "devtools", "roxygen2", "testthat")
+pkgs <- c("RCurl", "jsonlite", "statmod", "devtools", "roxygen2", "testthat", "R.utils")
```
Comment (michalkurka) [inline README.md:280]: what is it needed for? we should know ;)

Comment (satai) [reply README.md:280]: When I start with a totally clean computer and install only the documented packages, I get

```> setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
Error in loadNamespace(name) : there is no package called ‘R.utils’
```

when runnning tests (`R -f runit_coxph_concordance_heart.r`)

Comment (michalkurka) [reply README.md:280]: The list of packages to run tests is longer, IMHO maybe the guide should only list the packages needed to build without tests. For running tests, there is a utility function that installs all required packages.

Comment (satai) [reply README.md:280]: OK, I will make it clear in README...

**Pull request closed without merging**